presto-cassandra/pom.xml

```diff
@@ -305,10 +305,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <!-- integration tests take a very long time so only run them in the CI server -->
-                    <excludes>
-                        <exclude>**/TestCassandraDistributed.java</exclude>
-                    </excludes>
+                    <threadCount>1</threadCount>
```

Comment:
Are we ok with adding yet another Travis executor? What about moving these to presto-checks?